static/app/views/performance/landing/widgets/widgets/lineChartListWidget.tsx

```diff
@@ -156,7 +156,16 @@ export function LineChartListWidget(props: Props) {
         }
         return (
           <EventsRequest
-            {...pick(provided, ['children', 'organization', 'yAxis'])}
+            {...pick(provided, [
+              'children',
+              'organization',
+              'yAxis',
+              'period',
+              'start',
+              'end',
+              'environment',
+              'project',
+            ])}
```

Comment:
Nit: having this list as a constant somewhere might be useful

Reply:
True